backend/src/main/java/org/eclipse/tractusx/puris/backend/delivery/logic/service/DeliveryService.java

```diff
@@ -136,23 +136,22 @@ protected List<String> basicValidation(Delivery delivery) {
         List<String> errors = new ArrayList<>();
 
         if (delivery.getQuantity() < 0) {
-            errors.add("Quantity must be greater than or equal to 0.");
+            errors.add(String.format("Quantity '%d'must be greater than or equal to 0.", delivery.getQuantity()));
```

Comment:
```suggestion
            errors.add(String.format("Quantity '%d' must be greater than or equal to 0.", delivery.getQuantity()));
```

sorry for maybe sneaking in that typo :/